sql/migrations/series/up.sql

```diff
@@ -0,0 +1,56 @@
+BEGIN;
+
+ALTER TYPE bookbrainz.entity_type ADD VALUE 'Series';
```

Comment:
```suggestion
ALTER TYPE bookbrainz.entity_type ADD VALUE IF NOT EXISTS 'Series';
```

Since we don't delete the type in down.sql, this allows us to run up.sql again after running down.sql